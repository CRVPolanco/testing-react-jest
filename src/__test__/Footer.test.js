import React from 'react';
import { mount, shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../components/Footer';

describe('footer test', () => {
  const footer = shallow(<Footer />);

  test('footer test start', () => {
    expect(footer.length).toEqual(1);
  });
  test('title render', () => {
    expect(footer.find(".Footer-title").first().text()).toEqual("Kajlo Store");
  })
});

describe('footer snapshot', () => {
  test('check component ui if does not change ', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  })
});
