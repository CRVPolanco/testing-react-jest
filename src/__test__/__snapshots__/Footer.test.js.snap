// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer snapshot check component ui if does not change  1`] = `
<div
  className="Footer class"
>
  <p
    className="Footer-title"
  >
    Kajlo Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
